client/my-sites/checkout/controller.jsx

```diff
@@ -288,19 +286,16 @@ export function upsellNudge( context, next ) {
 }
 
 export function upsellRedirect( context, next ) {
-	const { receiptId, site, upsellMeta, upsellType } = context.params;
+	const { receiptId, site /*, upsellMeta, upsellType */ } = context.params;
 
 	setSectionMiddleware( { name: 'checkout-offer-redirect' } )( context );
 
 	let upsellExperimentName;
 	let upsellExperimentAssignmentName;
 	let upsellUrl;
 
-	if ( PROFESSIONAL_EMAIL_OFFER === upsellType ) {
```

Comment:
Since we no longer use that constant we could remove its definition:

```javascript
export const PROFESSIONAL_EMAIL_OFFER = 'professional-email-offer';
export type SupportedUpsellType = typeof PROFESSIONAL_EMAIL_OFFER;
```

Reply:
Good catch -- I've commented the lines out, but added some explanatory comments so future developers can use a similar pattern if they so choose.